README.md

```diff
@@ -193,124 +200,167 @@ Other acceptable location examples:
 
-Read more about the [Globalping Probe](https://github.com/jsdelivr/globalping-probe) in the dedicated repo.
+This ensures that popular locations, like `from Amsterdam` or `from AWS`, are automatically load-balanced across multiple probes within the same location, preventing the overload of specific probes.
 
+#### Test result variability
+When requesting a specific number of probes, there is no guarantee that the API will respond with the exact amount.
 
-## Limits | WIP
+#### Probe availability rules
+Globalping exposes and lets you interact with probes that are currently **online**. 
 
-Our platform has multiple limits to avoid abusive behaviour and at the same time motivate people to contribute to the sustainability of our platform.
+Here's when a probe is considered online and available through our API:
+- **It's up-to-date.** A probe running an older version will be forced to auto-update before it becomes available.
+- **It passes quality control tests.** Every probe must pass a test, ensuring it's connected to a reasonably stable network without packet loss. This test is done regularly; if a probe fails, it gets taken offline.
 
-#### Global limits
+This whole process is completely automated and managed by our platform.
 
-These limits are applied per IP address regardless if an API key supplied or not.
+## Join the network – Run a virtual probe
+Globalping relies on the community to expand its probe network. While we maintain our own probes in key locations, we welcome any support from both corporate partners and individuals. 
+[Join our network](https://github.com/jsdelivr/globalping-probe) and help make the internet faster for everyone by running a virtual probe (or several).
 
-- 100 POST requests per minute per IP. No GET limits are implemented to support "real-time" use-cases.
-- A single measurement is limited to 200 probes per location and 500 total probes
+### Quick start guide
+You can run a Globalping probe on any machine that can run a Docker container, supporting x86 and ARM architectures. For example, you can use a rented VPS, a dedicated server with spare capacity, or even your locally hosted Raspberry PI.
 
+Use this command to create and run the Globalping probe container:
 
-#### Un-authenticated users
+```
+docker run -d --log-driver local --network host --restart=always --name globalping-probe ghcr.io/jsdelivr/globalping-probe
+```
+Find instructions for Podman [here](https://github.com/jsdelivr/globalping-probe#podman-alternative) 
 
-Anybody can connect to our API and start using it with no credentials required.
-In this case we limit the amount of tests an IP address can run. A single test is defined as a succesful measurement we run and return to the user.
-A limit of 10 tests means the user can run either 10 measurements with the probe limit set to 1 per measurement, or a single measurement with the probe limit set to 10.
+### Notes on probe security and customization
```

Comment:
Please revert back to the original and make smaller changes if needed. Currently every point has lot some kind of important information. And lets not add a bold header, its not a big focus, people should be able to skim and skip if they want